docs/manual/manual_dronecan.rst

```diff
@@ -44,11 +44,274 @@ Servo modules do not support DroneCAN, as shown in the table below.
+software and hardware version of the node, and the name of the node. For Vertiq modules, the name of each node is currently “iq_motion.esc”.
+
+Refer to the `uavcan.protocol.GetNodeInfo section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#getnodeinfo>`_ in the specification 
+for more details
+
+uavcan.protocol.param.GetSet (Data Type ID = 11)
+##################################################
+Used to get or set the value of a configuration parameter using either the index or the name of the parameter. The configuration parameters currently available on standard 
+Vertiq modules are listed in the :ref:`dronecan_configuration_parameters` section below. The request message should contain the index of the parameter or the name of the parameter depending 
+on how you are accessing it, and it should contain a value to set the parameter to if you are setting it. **Parameter indices are not guaranteed to remain consistent across 
+firmware versions. Indices should only be used for parameter discovery, when accessing the parameter directly it is recommended to always use the name**
+
+The response message will contain the value of the parameter, information about its default, minimum, and maximum values, and the name of the parameter.
+
+Refer to the `uavcan.protocol.param.GetSet section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#getset>`_ in the specification 
+for more details
+
+uavcan.protocol.RestartNode (Data Type ID = 5)
+###############################################
+This request has one field, its “magic number”. This field is an unsigned 40 bit integer. The magic number is used to identify that this is an intentional restart request, 
+and we have also extended it to allow for two different types of reboots. One magic number performs a normal reboot that simply restarts into the normal application firmware, 
+and the other reboots the device into the ST bootloader, which allows it to be programmed with new firmware. The reboot-into-bootloader can be useful for firmware updates. 
+The numbers for each are:
+
+* Normal Reboot Magic Number = 0xACCE551B1E
+* Reboot to Bootloader Magic Number = 0xBEEFCAFE
+
+The response to this request contains one boolean field. This field will be true if the number received was a valid magic number and the motor is restarting, and false if the 
+number received was not one of the valid numbers.
+
+Refer to the `uavcan.protocol.RestartNode section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#restartnode>`_ in the specification 
+for more details.
+
+uavcan.protocol.file.BeginFirmwareUpdate (Data Type ID = 40)
+#############################################################
+On modules that support DroneCAN firmware updates, this command causes the module to enter into upgrade mode, and begin attempting to firmware update over DroneCAN. 

```

Comment:
do we have a list of those that do? Or do all motors that support dronecan have this?

Reply:
Long-term I want to cover in that a new section of the manual: https://app.asana.com/0/1201945089139714/1204392191422922/f

Though right now all of our stock modules that support DroneCAN do support the upgrade as well